xml-parsing-example/Makefile

```diff
@@ -0,0 +1,7 @@
+run-example: build
+	docker run --rm -ti --name example -v$(PWD):/source example /fluent-bit/bin/fluent-bit -c flb.conf
```

Comment:
You probably need to remove the -i as it will fail if not a tty, like in most ci.